### ISSUE TITLE
Tie python tests into CIT

### ISSUE THREAD

**Issue by mcmasterg** [SymbiFlow/prjxray#56]
There are some test_* things in tools/ that should ideally be run as part of submit check

**Comment by mithro** [comment]
@elmsfu - Is this done now?

**Comment by elmsfu** [comment]
> @elmsfu - Is this done now?

Not to my knowledge SymbiFlow/symbiflow-arch-defs#520 runs tests under symbiflow-arch-defs/utils.

We should consider gathering coverage information.

**Comment by mithro** [comment]
Hrm, the Python tests do seem to be running?
```
if [ -e env/bin/activate ]; then . env/bin/activate; fi; which py.test; py.test --ignore third_party --ignore .git --ignore env --ignore build --ignore docs/venv --ignore fuzzers --ignore minitests --ignore experiments --doctest-modules --junitxml=build/py_test_results.xml
/home/travis/build/SymbiFlow/prjxray/env/bin/py.test
============================= test session starts ==============================
platform linux -- Python 3.6.3, pytest-4.4.0, py-1.8.0, pluggy-0.9.0
rootdir: /home/travis/build/SymbiFlow/prjxray
collected 15 items                                                             
docs/markdown_code_symlinks.py .                                         [  6%]
prjxray/lib.py .                                                         [ 13%]
prjxray/util.py .                                                        [ 20%]
tests/test_util.py .                                                     [ 26%]
utils/cmp.py .                                                           [ 33%]
utils/sort_db.py .........                                               [ 93%]
utils/xjson.py .                                                         [100%]
```

**Comment by elmsfu** [comment]
Closing. Looks like you fixed it in #491
